src/components/Tools/ColorPicker/index.tsx

```diff
@@ -0,0 +1,179 @@
+import { useEffect, useState } from 'react'
+import { FaClipboard, FaDownload, FaEyeDropper, FaTrash } from 'react-icons/fa'
+import { useBodyHiding } from '@/hooks/useBodyHiding'
+import type { Color } from './types'
+import { copyToClipboard, exportColorsToFile, hexToFormats } from './utils'
+import {
+  addColorToHistory,
+  clearColorHistory,
+  loadColorHistory,
+  saveColorHistory,
+} from './storage'
+
+function ColorPicker() {
+  const [currentColor, setCurrentColor] = useState<Color | null>(null)
+  const [history, setHistory] = useState<Color[]>([])
+  const [isPicking, setIsPicking] = useState(false)
+
+  // Hide the extension popup while waiting for user color selection
+  useBodyHiding(isPicking)
+
+  // Load history on mount
+  useEffect(() => {
+    loadColorHistory().then(setHistory)
+  }, [])
+
+  // Save history
+  const saveHistory = (colors: Color[]) => {
+    setHistory(colors)
+    saveColorHistory(colors)
+  }
+
+  // Pick color using EyeDropper API
+  const pickColor = async () => {
+    try {
+      setIsPicking(true)
+
+      const eyeDropper = new (window as any).EyeDropper()
+      const result = await eyeDropper.open()
+
+      const color = hexToFormats(result.sRGBHex.toUpperCase())
+      setCurrentColor(color)
+
+      const newHistory = addColorToHistory(color, history)
+      saveHistory(newHistory)
+    }
+    catch {
+      // User cancelled
+    }
+    finally {
+      setIsPicking(false)
+    }
+  }
+
+  // Clear history
+  const handleClearHistory = () => {
+    setHistory([])
+    setCurrentColor(null)
+    clearColorHistory()
+  }
+
+  return (
+    <div className="p-1 space-y-3">
+      {/* Pick Color Button */}
+      <button
+        onClick={pickColor}
+        className="w-full flex items-center justify-center gap-2 bg-black hover:bg-gray-800 text-gray-400 hover:text-white py-2 px-4 rounded-lg transition-colors shadow-md shadow-neutral-400/70"
```

Comment:
It's probably better to preserve the previous dark buttons to maintain consistency across the extension:

<img width="451" height="348" alt="button_color" src="https://github.com/user-attachments/assets/83dcc29c-726b-419b-b422-1a3c86790258" />